knowledge-base/troubleshoot-mac-proxy-settings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Troubleshoot Mac Proxy Settings
+description: Troubleshooting Mac Proxy Settings will help isolate Issues
+type: troubleshooting
+page_title: Verifying Mac Proxy Settings
```

Comment:
Note: If possible, try to use different words than the title...but if not, no worries

Reply:
Not sure I can use different words for Mac Proxy Settings. This will probably stay as is.